### ISSUE TITLE
Exclamation mark "!" problem during training

### ISSUE THREAD

**Issue by gitup-l** [dvlab-research/Seg-Zero#60]
Your work is very helpful to me. However, when I used 4 H20 graphics cards with 96G memory to train my own data set on the 7B model, I encountered a situation where the output suddenly became all exclamation marks and could not be restored. Did you encounter this during the training process?

![Image](https://github.com/user-attachments/assets/4f434015-92b7-4656-aa8b-902036ce469e)

**Comment by LiuRicky** [comment]
Thanks for your interest. 

Could you provide more context to your problem? For example, please provide an example of your case.

**Comment by gitup-l** [comment]
I am working on intention-related work. 7B's Visonreasoner actually does a great job on my work, but some complex problems need to be fine-tuned. I retrained for a total of 73 steps. There are 73 steps in total for retraining. At first everything was normal and the sore was between 4 and 6. Around the 60th epoch, there was a problem that neither the thought chain nor the segmentation prompt was output, only the exclamation mark was output. I don't know if this is clear enough, thank you.

**Comment by gitup-l** [comment]
Thanks for your reply. 
The evaluation after 60 steps is also abnormal. I reduced the learning rate from e-6 to e-7. This method avoids the previous problem but the training effect will be greatly reduced.
